workloads/whereabouts/node-density.yml

```diff
@@ -0,0 +1,36 @@
+---
+global:
+  gc: {{.GC}}
+  gcMetrics: {{.GC_METRICS}}
+  indexerConfig:
+    esServers: ["{{.ES_SERVER}}"]
+    insecureSkipVerify: true
+    defaultIndex: {{.ES_INDEX}}
+    type: opensearch
+  measurements:
+    - name: podLatency
+      thresholds:
+        - conditionType: Ready
+          metric: P99
+          threshold: 2000ms
```

Comment:
We know multus cannot achieve this, should we consider higher threshold here?

Reply:
Good point, how much do we want to allow for "multus overhead"? I would be okay with adding a whole second for it. Do we want to throw a question toward the multus team?

Reply:
To me, the answer to this question comes from the results you've observed. If you don't know, then I would omit the threshold here instead.

Reply:
I think having a threshold here has value, cause the multus / whereabouts team can have the joy of seeing a test go from red to green.